webpack.prod.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const fs = require('fs');
+const { createConfig } = require('@edx/frontend-build');
+
+/** This condition confirms whether the configuration for the MFE has switched to a JS-based configuration
+ * as previously implemented in frontend-build and frontend-platform. If the environment variable exists, then
+ * an env.config.js file will be created at the root directory and its env variables can be accessed with getConfig().
+ *
+ * https://github.com/openedx/frontend-build/blob/master/docs/0002-js-environment-config.md
+ * https://github.com/openedx/frontend-platform/blob/master/docs/decisions/0007-javascript-file-configuration.rst
+ */
+if (process.env.JS_CONFIG_FILEPATH) {
+  fs.copyFile(process.env.JS_CONFIG_FILEPATH, 'env.config.js', (err) => {
+    if (err) { throw err; }
+  });
+}
```

Comment:
Great suggestion! I think if we end up going with this implementation, then we should definitely consider moving it upstream. For the sake of stating the obvious and making sure I know the _why_, similarly with the `webpack.config.js` issue I already made [in frontend-build](https://github.com/openedx/frontend-build/issues/513), this would mean that teams wouldn't have to create a custom webpack config with this logic or modify their already-existing configs _per repo_.

Reply:
Alright, I removed the custom configurations in webpack prod/dev and tested them through the various permutations of keeping/removing `.env.*` and `env.config.js`, as well as with the local production build using Tubular. All worked as before, and the only things kept in those configs now are related to handling `env.config.js`
Thanks for bringing these up!